main.js

```diff
@@ -173,6 +178,8 @@ var start_local_server = function() {
       serverRunning = false;
     });
     subpy.unref();
+  } else {
+    mainWindow.webContents.executeJavaScript("console.log('Unable to find OpenBazaar-Server at: '" + serverPath + "')");
```

Comment:
Even if it works fine here, it's a best practice and helps head off future bugs.

That, and es-lint should have caught it. Is es-lint not working for you on commit?